### DISCUSSION TITLE
[Descriptions ] Want Descriptions to add responsive layout

### DISCUSSION THREAD

**Discussion by journeybyte** [element-plus/element-plus#13570] [Feature Request]
Feature type: **`New Component`**

## Description

Want Descriptions to add responsive layout

![image](https://github.com/element-plus/element-plus/assets/50314536/df27fed9-c5c6-4a12-b12f-80b99acc89b0)

**Comment by kaine0923** [comment]
你可以监听页面的大小改变给`column`属性不同的值，默认是3，所以你截图他会溢出。
